core/src/main/resources/reference.conf

```diff
@@ -184,11 +194,17 @@ system {
     }
 
     gcs {
+      # `gcs.parallelism` governs GCS *batch operation* parallelism. Non-batch GCS operations like 'write' or
+      # 'read lines' are processed as nio commands whose parallelism is governed by `nio.parallelism` below.
       parallelism = 10
+      # Batches of non-zero size will be sent to Google with at most `max-batch-size` commands and not having
+      # more than `max-batch-timespan` between when the first and last command was batched.
+      max-batch-size = 100
+      max-batch-timespan = 5 seconds
     }
 
     nio {
-      parallellism = 10
+      parallelism = 10
```

Comment:
Again, at least needs to be in release notes. See also: `HTTP_REFERER`

Reply:
I can mention in the release notes that `parallelism` used to be misspelled in `reference.conf`, though I don't think fixing the spelling here introduces a change in behavior. If a user had misspelled the word in config Cromwell would have ignored it and [fallen back on the defaults](https://github.com/broadinstitute/cromwell/blob/9a207229343433a42aab678566bd552323e1f6b4/engine/src/main/scala/cromwell/server/CromwellRootActor.scala#L108), while if it was spelled correctly it should have just worked.